extensions/kafka-client/deployment/src/main/java/io/quarkus/kafka/client/deployment/DevServicesKafkaProcessor.java

```diff
@@ -231,17 +229,36 @@ private KafkaBroker startKafka(KafkaDevServiceCfg config,
 
         // Starting the broker
         final Supplier<KafkaBroker> defaultKafkaBrokerSupplier = () -> {
-            RedPandaKafkaContainer container = new RedPandaKafkaContainer(
-                    DockerImageName.parse(config.imageName),
-                    config.fixedExposedPort,
-                    launchMode.getLaunchMode() == LaunchMode.DEVELOPMENT ? config.serviceName : null,
-                    useSharedNetwork, config.redpanda);
-            timeout.ifPresent(container::withStartupTimeout);
-            container.start();
-
-            return new KafkaBroker(
-                    container.getBootstrapServers(),
-                    container::close);
+            if (config.imageName.contains("strimzi")) {
```

Comment:
Initially, I did it very strict (I realise I forgot to remove the pattern from this class). 
Then I thought It would be useful to run with a locally built custom strimzi image for testing purposes. 
BTW maybe it is important to note that it is not possible to customize the Kafka broker properties for the dev service.